integration-tests/smoke/ccip_test.go

```diff
@@ -83,7 +84,13 @@ func TestInitialDeployOnLocal(t *testing.T) {
 			require.NoError(t, err)
 			block := latesthdr.Number.Uint64()
 			startBlocks[dest] = &block
-			seqNum := ccdeploy.TestSendRequest(t, e, state, src, dest, false, nil)
+			seqNum := ccdeploy.TestSendRequest(t, e, state, src, dest, false, router.ClientEVM2AnyMessage{
```

Comment:
nit/ what about a `TestSend(Dummy|Default)Request`? I could convenient to have it so that we won't deal with creating those fields if we are not going to use them.

Reply:
Yeah, I thought about adding that but I think we wanna see how the usage of these funcs evolves and build abstractions after we have many use cases. I think for now this is the most flexible, and can improve further.